### PR TITLE
Columbia Escort Frigate

### PR DESCRIPTION
**Pull request by SergeiKoralev**
This heavily modified Kelvin Class ship is meant to either supplement a main battle fleet in the post Wolf 359 Alpha Quadrant, or to serve as a primary escort or security ship for a convoy or escort. The  Columbia boasts four decks with all required necessities for a Starfleet ship of war while still enjoying some of the creature comfort and research capability of her larger or more antiquated counterparts. It even contains a stigma not frequently found on other starfleet ships. A cloning bay, although rarely used, does exist and is available for use to those crew members who are not revolted by the thought.

### CODE REVIEW

**Comment by alexkar598** [comment]
Note that you can integrate images in your post by using the syntax ```` example: 

````

I changed it for you this time

File: DeepSpace13.dme [hunk 1]
```diff
@@ -1,2821 +1,2819 @@
...
-#include "code\modules\antagonists\clockcult\clock_items\clockwork_slab.dm"
-#include "code\modules\antagonists\clockcult\clock_items\clockwork_weaponry.dm"
-#include "code\modules\antagonists\clockcult\clock_items\construct_chassis.dm"
-#include "code\modules\antagonists\clockcult\clock_items\integration_cog.dm"
-#include "code\modules\antagonists\clockcult\clock_items\judicial_visor.dm"
-#include "code\modules\antagonists\clockcult\clock_items\replica_fabricator.dm"
-#include "code\modules\antagonists\clockcult\clock_items\soul_vessel.dm"
-#include "code\modules\antagonists\clockcult\clock_items\wraith_spectacles.dm"
-#include "code\modules\antagonists\clockcult\clock_items\clock_weapons\_call_weapon.dm"
-#include "code\modules\a
```
Comment (Crossedfall) [inline DeepSpace13.dme]: This file should be removed from the PR unless actual changes to the DME are needed.

Comment (Crossedfall) [reply DeepSpace13.dme]: Resolved now.

**Comment by ben10083-zz** [comment]
For the deck with the bridge, I recommend moving everything down one or two (area below the lifts) to allow for a passage between the sides of the deck without requiring bridge access.

**Comment by alexkar598** [comment]
Is this PR still a WIP or is it ready for review?

**Comment by SergeiKoralev** [comment]
> For the deck with the bridge, I recommend moving everything down one or two (area below the lifts) to allow for a passage between the sides of the deck without requiring bridge access.

For deck 1 the primary role is bridge, AI, and t-comms. (Thinking of that I'll be adding CE access to the AI doors so CE can get to t-comms.) If someone needs to be able to access the exact same equipment on the other side of deck 1 and doesn't have bridge access they can go around via deck 2.

**Comment by SergeiKoralev** [comment]
> Is this PR still a WIP or is it ready for review?

It's ready for review, but the idea of a map that isn't a Work in progress is a myth. Any tweaks from here on out will be covered in fairly good detail through the commit logs. See above reply to Ben for details on what will probably be the next commit.

**Comment by Michael-Ashfield** [comment]
I like this, seems like an interesting and fun ship, I just feel theres abit too much carpet really. I imagine areas like engineering would have some more fitting floor, such as tiling, since it is meant to be a messy, possibly dangrous, working area. This also goes for the kictchen and external airlocks, which don't seem suitable for nice carpet. Also different floors add more variety to the look and feel of different areas of the ship.

**Review by francinum — CHANGES_REQUESTED** [review]
Missing blobstart, carpspawn landmarks.

**Comment by Harry-Jubilee** [comment]
@Michael-Ashfield This is a federation starship... most of it has fancy carpet down

**Comment by SergeiKoralev** [comment]
> Missing blobstart, carpspawn landmarks.

Intended.

**Review by Harry-Jubilee — APPROVED** [review]
All looks fantastic :)

**Review by Kmc2000 — APPROVED** [review]
No other maps have those landmarks

**Comment by francinum** [comment]
@Kmc2000 that sounds like an issue with the other maps. not this one.

**Pull request merged**